sklearn/inspection/tests/test_partial_dependence.py

```diff
@@ -62,12 +62,13 @@
 iris = load_iris()
 
 
+@pytest.mark.filterwarnings("ignore:A Bunch will be returned")
```

Comment:
Here this is a case, that we cannot avoid the warning and filtering is meaningful

Reply:
Great. Left intact.